mix.exs

```diff
@@ -1,27 +1,14 @@
-defmodule Mix.Tasks.Compile.Exfswatch do
```

Comment:
To clarify, the reason i think it's better is that I would expect compilation of C code to happen during a mix compile step but not during normal runtime.